src/atoms/rangeInput/RangeInput.tsx

```diff
@@ -190,9 +190,13 @@ export const RangeInput = ({
               touched ? classes.showThumb : classes.hideThumb
             }`}
             onChange={handleValueChange}
-            onFocus={() => {
-              setTouched(true)
-              onTouched(true)
+            onFocus={(e) => {
+              if (!touched) {
+                handleValueChange(e)
+              } else {
```

Comment:
Then this `else` doesn't make sense, because `handleChange` will always set `touched` to `true` :-)